data/Remnant Cognizance.txt

```diff
@@ -0,0 +1,626 @@
+
+
+
+mission "Remnant: Cognizance 6"
+	name "Take Plume to Nasqueron"
+	description "Plume wants to witness first hand what is going on in Nenia."
+	source "Caelian"
+	destination "Nasqueron"
+	to offer
+		has "Remnant: Cognizance 5: done"
+	on offer
+		conversation
+			`In the starport Prefect Chilia and Plume are gesturing at some details on a datapad when you approach. Chilia notices you first and waves you over. "Thank you for joining us. While I put together some more long-term plans, could you take Plume to Nasqueron?"`
+			choice
+				`	"Why do we need to go back?"`
+				`	"Let's go."`
+					goto action
+			`	"Recordings and sensor readings are useful data, but they don't really compare to actually seeing what is going on and experiencing the data first-hand." says Plume. "There aren't too many opportunities to do it in my line of work - especially when my expertise is needed by others who would rather not spend their time watching wildlife. But I digress."`
+			label action
+			`	With a respectful bow to Chilia (who returns it respectfully), Plume heads for your ship. "I will be waiting onboard for you, Captain."`
+				accept
+	npc
+		government "Korath"
+		personality staying
+		system "Nenia"
+		fleet
+			names "korath"
+			variant
+				"Korath Raider (Ember)" 2
+	on complete
+		conversation
+			`The descent into the clouds of Nasqueron is becoming almost familiar to you. The whispy layers giving way to more volumous formations and the ocasional turbulent storm. Beside you, Plume is eagerly peering out the window, and you remind yourself that this may well be his first time actually seeing void spries on their world.`
+			`	You reach the altitude that has had the most void sprites on it in the past, and activate the sensors. Plume looks around, peering out the windows. "Where are they? aren't there usually some around? They always seemed plentiful in the recordings." You focus on the sensors, and after a few minutes pick up some distant contacts. You follow them down into the lower atmosphere and eventually start seeing more void sprites."`
+			`	"That's close enough, captain. I'll go outside and watch them from the hull." You help him into the pressure suit and cycle the airlock behind him. A brief feeling of dissapointment flares within you, but you quickly suppress it. "This is, after all, Plume's job" you tell yourself as you settle back into your chair to wait for Plume to return.`
+
+
+
+mission "Remnant: Cognizance 7"
+	name "Return Plume to Caelian"
+	description "Plume wants to check his records of the research on void sprites to verify some information."
+	landing
+	source "Nasqueron"
+	destination "Aventine"
+	to offer
+		has "Remnant: Cognizance 6: done"
+	on offer
+		conversation
+			`After several hours Plume returns to the ship, spurred by the alarms in the suit warning him about the air levels. As you help him out of the suit he looks like he is having trouble deciding if he should be thrilled or worried.`
```

Comment:
`After several hours Plume returns to the ship, spurred by the alarms in the suit warning him about his air levels. As you help him out of the suit he looks like he is having trouble deciding if he should be thrilled or worried.`

Reply:
Fixed.